addons/website/views/snippets/s_opening_hours.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_opening_hours" name="Opening Hours">
+    <section class="s_opening_hours">
+        <div class="container-fluid">
+            <div class="row o_grid_mode" data-row-count="13">
+                <div class="o_grid_item o_grid_item_image g-col-lg-13 g-height-12 col-lg-12" style="z-index: 1; grid-area: 1 / 1 / 14 / 13; --grid-item-padding-y: 0px; --grid-item-padding-x: 0px;">
+                    <img class="img img-fluid" src="/web/image/website.s_cover_default_image" alt=""/>
+                </div>
+                <!-- This next div is intended to correct a visual issue, specifically a vertical line anomaly next to the hours, that appears only in the modal preview.-->
+                <div class="o_grid_item o_cc o_cc1 g-col-lg-10 g-height-7 col-lg-10 s_dialog_preview" style="z-index: 2; grid-area: 7 / 2 / 14 / 12; --grid-item-padding-y: 0px; --grid-item-padding-x: 0px;"/>
+                <div class="o_grid_item g-col-lg-6 g-height-7 col-lg-6 o_cc o_cc1" style="grid-area: 7 / 2 / 14 / 8; z-index: 3; --grid-item-padding-y: 32px; --grid-item-padding-x: 40px;">
+                    <h2 class="display-4-fs" style="text-align: left;">Let’s get in touch</h2>
```

Comment:
Is there a reason you declare `style="text-align: left;"` here but not on days + hours text ? I guess if it's default, we can get rid of this inline style ? Same for the `lead` paragraph